doc/manual/manual.tex

```diff
@@ -8485,23 +8488,249 @@ \subsubsection{The van Keken thermochemical composition benchmark}
+shown below.
+
+\lstinputlisting[language=prmfile]{cookbooks/benchmarks/vof-van-keken/variation.part.prm.out}
+
+\begin{figure}[htb]
+    \centering
+    \includegraphics[width=0.5\textwidth]{cookbooks/benchmarks/vof-van-keken/init_diff_rms_vel_comparison.png}
+    \caption{\it Computations of the Van Keken problem made with the VOF
+        interface tracking algorithm showing the evolution of the RMS velocity
+        as a function of time for small changes in the amplitude $a$ of the
+        cosine function in the initial condition at 7 levels of refinement.
+        Compare to Figures~\ref{fig:vk-6} and~\ref{fig:vof-vk-1}.
+    }
+    \label{fig:vof-vk-3}
+\end{figure}
+
+In these computations we vary the value of $a$ from its usual value of $a = 0.02$ to 
+$5\% = 0.001$ below its usual value to $5\%$ above its usual value in increments of $0.01$.
+In other words, we compare the values for $a  =0.019$, $0.020$, and $0.021$.
+Upon examination of Figure~\ref{fig:vof-vk-3}, we see a visible variation in the location of 
+the second peak, although the overall shape of the curve remains consistent with the curves 
+in Figure~\ref{fig:vof-vk-1}.
+The size of this variation in the initial conditions cannot be expected to be reproduced using 
+the standard compositional quadrature initialization procedure for VOF unless the cell size is on the scale 
+of the change in the value of $a$; i.e., $h \, \lessapprox \, \Delta a = 0.001$.
+We also note that the smoothing parameter, which would produce a
+$10^{-3}\leq C \leq 1 - 10^{-3}$ band on the order of the same size as the amplitude 
+variation shown here would be approximately $2.8 \cdot 10^{-4}$.
+This perturbation is much smaller than any of the changes in width of the smoothed regions in 
+the computations shown in Figure~\ref{fig:vk-6}.
+In summary, these results demonstrate the sensitivity of the discontinuous version of the van 
+Keken problem to even extremely small variations in the initial conditions.
+
 \subsubsection{The Rayleigh-Taylor instability}
 \label{sec:benchmark-rayleigh-taylor}
 
 \textit{This section was contributed by Cedric Thieulot.}
 
 This benchmark is carried out in \cite{Deu08,Ger10,thie11} and is 
-based on the analytical solution by Ramberg\cite{ramb68}, 
+based on the analytical solution by Ramberg \cite{ramb68}, 
 which  consists of a gravitationally unstable two-layer system.
 Free slip are imposed on the sides while no-slip boundary conditions are imposed on the
 top and the bottom of the box.
 Fluid 1 $(\rho_1,\eta_1)$ of thickness $h_1$ overlays 
 fluid 2 $(\rho_2,\eta_2)$ of thickness $h_2$ (with $h_1+h_2=L_y$).
 An initial sinusoidal disturbance of the interface between these
-layers is introduced and is characterised by an amplitude $\Delta$ and a
+layers is introduced and is characterized by an amplitude $\Delta$ and a
 wavelength $\lambda=L_x/2$ as shown in Figure~\ref{fig:RTi_setup}.
 
-\begin{figure}
+\begin{figure}[h!]
```

Comment:
Left due to possible relevance to correct figure behavior.

Reply:
What do you mean by `possible relevance`? Did you have to add this to create a correct figure, or can it be removed without changes? I would prefer to only add changes that are necessary to resolve issues.

Reply:
Let's not use `[h!]`. It completely messes up latex's placement algorithm. The default is exactly what we should be using instead. In other words, please revert this change.